config/routes.rb

```diff
@@ -23,6 +23,12 @@
   get '/auth/developer/callback' => 'dfe_sign_in#bypass_callback'
   get '/auth/dfe/sign-out' => 'dfe_sign_in#redirect_after_dsi_signout'
 
+  get '/auth/onelogin/callback', to: 'omniauth_callbacks#complete'
```

Comment:
The naming convension for this is usually something like
```suggestion
  get '/auth/onelogin/callback', to: 'omniauth_callbacks#onelogin'
```
This allows applications to support multiple Omniauth providers